plugins/discourse-local-dates/assets/javascripts/initializers/discourse-local-dates.js.es6

```diff
@@ -138,9 +140,54 @@ function buildHtmlPreview(element, siteSettings) {
   previewsNode.classList.add("locale-dates-previews");
   htmlPreviews.forEach((htmlPreview) => previewsNode.appendChild(htmlPreview));
 
+  previewsNode.appendChild(_downloadCalendarNode(element));
+
   return previewsNode.outerHTML;
 }
 
+function _downloadCalendarNode(element) {
```

Comment:
I noticed this initializer is getitng quite big. We might want to break it up into another file at some point.